gstcefsrc.cc

```diff
@@ -2,12 +2,8 @@
 #include <iostream>
 #include <sstream>
 
-#include <include/cef_app.h>
```

Comment:
I can't tell if the modified includes have any impact, but they shouldn't be part of this commit in any case :)

Reply:
These are already included in the header. I'll split out into another commit.